modules/web-console/e2e/testcafe/fixtures/configuration/basic.js

```diff
@@ -34,6 +34,17 @@ fixture('Basic configuration')
     })
     .after(dropTestDB);
 
+test('Check selector picker version list', async(t) => {
+    const knownSupportedVersions = ['Ignite 2.7', 'Ignite 2.6', 'Ignite 2.5', 'Ignite 2.4', 'Ignite 2.3', 'Ignite 2.1', 'Ignite 2.0', 'Ignite 1.x'];
+    const page = new PageConfigurationBasic();
+
+    await t.hover(page.versionPicker._selector)
+        .expect(page.versionPicker.menuItems.count).eql(knownSupportedVersions.length, 'Should contains 8 supported versions.');
```

Comment:
This assertion is redundant.

Reply:
Version picker should not contain unsupported versions. Changed error message.